src/Fixer/FunctionNotation/MethodArgumentSpaceFixer.php

```diff
@@ -187,4 +309,16 @@ private function isCommentLastLineToken(Tokens $tokens, $index)
 
         return $content !== ltrim($content, "\r\n");
     }
+
+    /**
+     * Checks if token is new line.
+     *
+     * @param Token $token
+     *
+     * @return bool
+     */
+    private function isNewLine(Token $token)
+    {
+        return strpos(trim($token->getContent(), " \r\t\0\x0B"), "\n") !== false;
```

Comment:
PHP CS Fixer actually support only `\n` and `\r\n` - we don't check other new-line combinations in rest of the tool.

`\r` was used by classic MacOS (before 2001 release of MacOs X)